### DISCUSSION TITLE
Is the ProductVersion property in a wixproj still necessary in wix v4?

### DISCUSSION THREAD

**Discussion by wmanning** [wixtoolset/issues#7365] [Questions] [answered]
Is it safe to remove the ProductVersion property in a wixproj in WiX v4?
Never really paid attention to it before; no idea what it does.

```<ProductVersion>3.6</ProductVersion>```

**Comment by robmen** [comment, marked as answer]
Yes. I _think_ HeatWave will remove it for you automatically if the value is a known "WiX 3" number (but I _could_ be imaging that we did that at FireGiant 😄).
